payu/models/fms.py

```diff
@@ -95,6 +120,8 @@ def collate(self):
         tile_fnames = [f for f in os.listdir(self.output_path)
                        if f[-4:].isdigit() and f[-8:-4] == '.nc.']
 
+        tile_fnames.sort()
```

Comment:
What is the sorting for?  (Just wondering, not important)

Reply:
The list of filenames isn't sorted at all, so unlike a glob is in pretty much random order. It makes it easier to visually scan the command that is echoed to stdout with this change, and it is what a user would expect to see IMO.